knative-operator/pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -174,6 +179,40 @@ func (a *ReconcileKnativeServing) installNetworkPolicies(instance *servingv1alph
 	return nil
 }
 
+// createConsoleCLIDownloadCR creates CR for kn CLI download link
+func (a *ReconcileKnativeServing) createConsoleCLIDownloadCR() error {
```

Comment:
Add an instance parameter so it can be treated as a "stage" and just for the sake of clarity, let's move the guts of this function, along with the accompanying `deleteConsoleCLIDownloadCR` function to their own package, creating a `cli/` subdir similar to `servicemesh/`. Within their own package, the function names could then be `Create` and `Delete`. That would keep the controller cleaner, I think. Make sense?

Reply:
Yes, thanks for the review!

I've updated the PR, also now the errors for create/delete of ConsoleCLIDownload are not suppressed but returned.